### ISSUE TITLE
MisconfigurationException in 2nd Tutorial (Classification)

### ISSUE THREAD

**Issue by MrPositron** [DiffEqML/torchdyn#43]
I am going over 2nd Torchdyn Tutorial and I encountered this error:
![image](https://user-images.githubusercontent.com/42044624/94368050-a16d0580-00fb-11eb-8dc8-6b4482c5bbca.png)

This error happens when I run this command:
![image](https://user-images.githubusercontent.com/42044624/94368129-16d8d600-00fc-11eb-9ad1-ee3793e9c312.png)

I think that this error is caused by PytorchLighting. However, I am a newbie to PytorchLighting. Thus, I don't know how to debug it

**Comment by Zymrael** [comment]
Hi @MrPositron, thank you for pointing this out. Could you let me know which versions of `torch` and `pytorch-lightning` you're currently using?

**Comment by MrPositron** [comment]
PyTorch version is 1.6. I ran it on colab. Furthermore, changing 
```
sched = {'scheduler': torch.optim.lr_scheduler.ReduceLROnPlateau(opt),
                 'monitor': 'loss', 
                 'interval': 'step',
                 'frequency': 10  }
```
to 
```
sched = {'scheduler': torch.optim.lr_scheduler.ReduceLROnPlateau(opt),
                 'monitor': 'train_loss', 
                 'interval': 'step',
                 'frequency': 10  }
```
does not result in error.

**Comment by Zymrael** [comment]
Make sure to have the metric you choose to `monitor` available to the `LightningModule` after each step. On my end the training loop works with both `loss` and `train_loss` (`pytorch-lightning==0.9.0`).

**Comment by Zymrael** [comment]
The new version of `pytorch-lightning` has fixed the issue, closing.
